packages/web-components/package.json

```diff
@@ -53,18 +53,18 @@
     "typescript": "~3.7.2"
   },
   "scripts": {
-    "build:prod": "node ./src/scripts/build-prod.js",
+    "build:prod": "node ./scripts/build-prod.js",
     "start:server": "serverless offline --out public/dist --stage local",
-    "start:dev": "node ./src/scripts/start-dev.js",
+    "start:dev": "node ./scripts/start-dev.js",
     "test:dev": "cross-env TZ=UTC jest --watch --color",
     "test:ci": "cross-env TZ=UTC jest --ci --colors --coverage --silent --forceExit",
     "lint": "concurrently \"tsc --noEmit\" \"eslint --cache --ext=ts,tsx,js,svelte src\"",
     "lint:fix": "eslint --cache --ext=ts,tsx,js,svelte src --fix",
     "fetch-config": "yarn config-manager fetchConfig web-components",
-    "build:serverless": "serverless webpack --out dist --stage dev",
-    "release:server:dev": "serverless deploy --stage dev",
+    "build:serverless": "node ./scripts/build-serverless.js",
+    "release:server:dev": "node ./scripts/release-serverless.js",
     "release:server:prod": "node ../../scripts/release/release-serverless.js web-components",
-    "release:npm:prod": "node ../../scripts/release/release-npm.js elements --skip-edit-release-note",
+    "release:npm:prod": "node ../../scripts/release/release-npm.js web-components --skip-edit-release-note",
```

Comment:
@willmcvay I think this will solve deployment issue of web-component release prod pipeline... Seems like typos in my previous commit in this file. That's great @vuhuucuong, if `release:npm:prod` works ok this time, please help me comment on ticket: https://github.com/reapit/foundations/issues/1107

Reply:
@phmngocnghia got you 👍